File: valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {

        char[] aray1 = s.toCharArray();
        char[] aray2 = t.toCharArray();

        Arrays.sort(aray1);
        Arrays.sort(aray2);
        
        String a = new String(aray1);
        String b = new String(aray2);
        if(a.equals(b))
        return true;
        return false;

    }
}

File: valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if(len(s) != len(t)):
            return False
        dicti = dict()
        for i in s:
          if(i not in dicti):
            dicti[i] = 1  
          else:
            dicti[i] += 1
        for i in t:
           if(i in dicti):
            dicti[i] -= 1
        for i in dicti:
            if(dicti[i] != 0):
                return False
        return True
